sos/report/plugins/foreman.py

```diff
@@ -326,6 +334,12 @@ class RedHatForeman(Foreman, SCLPlugin, RedHatPlugin):
     apachepkg = 'httpd'
 
     def setup(self):
+
+        self.add_file_tags({
+            '/var/log/foreman-installer/satellite.log.*': 'satellite_log',
```

Comment:
Maybe tagname `insataller_satellite_log` or `satellite_installer_log`? As under the term "the" Satellite log, I would imagine rather `/var/log-foreman-production.log`.

Reply:
This was to match the name Insights uses for it already. We could do `insights_satellite_log` and `satellite_installer_log` though just as easily.